### PR TITLE
[RFR] Add missing french translation keys

### LINKED ISSUE #1464: [react-admin@v2.0.0-beta1]Missing translation keys in ra-language-french
Some translation keys are missing in package ra-language-french ; it triggers some warnings in the console : 
- Warning: Missing translation for key: "ra.message.loading"
- Warning: Missing translation for key: "ra.page.loading"

**Environment**

* Admin-on-rest version: v2.0.0-beta1

### PR DESCRIPTION
**Pull request by alexisjanvier**
Fix #1464

### CODE REVIEW

File: packages/ra-language-french/index.js [hunk 1]
```diff
@@ -41,6 +48,8 @@ module.exports = {
             about: 'Au sujet de',
             not_found:
                 "L'URL saisie est incorrecte, ou vous avez suivi un mauvais lien.",
+            loading:
+                "La page est en cours de chargement, juste un moment s'il vous plaît.",
```
Comment (djhi) [inline packages/ra-language-french/index.js]: s/juste un moment s'il vous plaît/merci de bien vouloir patienter/

File: packages/ra-language-french/index.js [hunk 2]
```diff
@@ -62,6 +71,7 @@ module.exports = {
             updated: 'Élément mis à jour',
             created: 'Élément créé',
             deleted: 'Élément supprimé',
+            bad_item: 'Élément incorrect',
```
Comment (djhi) [inline packages/ra-language-french/index.js]: s/incorrect/inconnu/

Comment (djhi) [reply packages/ra-language-french/index.js]: btw, I think this one is missing in the english package too

Comment (alexisjanvier) [reply packages/ra-language-french/index.js]: Nope, it isn't https://github.com/marmelab/admin-on-rest/blob/next/packages/ra-language-english/index.js#L72

**Pull request merged**